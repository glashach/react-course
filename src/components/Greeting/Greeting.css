h1 {
  font-family: serif;
  font-size: 3em;
  color: salmon;
}

img {
  display: block;
}
