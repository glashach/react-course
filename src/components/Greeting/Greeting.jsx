import React from "react";
import "./Greeting.css";
import Logo from "../../assets/giphy.gif";

export const Greeting = () => (
  <>
    <h1>Hello, world!</h1>
    <img alt="Best greeting ever" src={Logo} />
  </>
);
