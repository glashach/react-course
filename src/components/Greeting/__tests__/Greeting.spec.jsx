import React from "react";
import renderer from "react-test-renderer";
import { Greeting } from "../Greeting";

describe("<Greeting />", () => {
  it("should match snapshot", () => {
    const component = renderer.create(<Greeting />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
