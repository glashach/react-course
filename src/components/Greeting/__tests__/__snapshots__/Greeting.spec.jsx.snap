// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Greeting /> should match snapshot 1`] = `
<h1>
  Hello, world!
</h1>
`;
